config/main.py

```diff
@@ -663,6 +663,17 @@ def _get_disabled_services_list(config_db):
 
     return disabled_services_list
 
+def _stop_all_services():
+    # stop all other services
+    for s in _get_sonic_services():
+        try:
+            clicommon.run_command("sudo systemctl stop {}".format(s))
+        except SystemExit as err:
+            click.echo("Error encountered while stopping service {}. Continuing..".format(s))
```

Comment:
Is it possible to capture the output returned by the subprocess and print that in the case of exception?

Reply:
Are you suggesting that for logging the error to syslog, we capture the output? Typically the error message goes on to the stdout and so does click.echo().

Reply:
Instead of the current message, I am suggesting to make it more descriptive  like "Error encountered while stopping service  swss.service, Systemd Message: 'Job for swss.service cancelled' Continuing..",